src/app/index.html

```diff
@@ -11,4 +11,7 @@
   <li>
     <a skyAppLink="/visual/edit-in-modal-grid">Edit it modal grid</a>
   </li>
+  <li>
+    <a skyAppLink="/visual/master-detail-grid">Master/Detail Grid</a>
```

Comment:
Looks like you might have forgot to check this file in? Getting a 404.

Reply:
Nope. That was removed due to the enterprise issues (as was discussed in flex time). I've fixed this.